tools/autograd/derivatives.yaml

```diff
@@ -571,9 +571,9 @@
 - name: geqrf(Tensor self) -> (Tensor a, Tensor tau)
   self: not_implemented("geqrf")
 
-- name: ger(Tensor self, Tensor vec2) -> Tensor
-  self: grad.mv(vec2.conj())
-  vec2: grad.t().mv(self.conj())
+- name: outer(Tensor self, Tensor vec2) -> Tensor
+  self: grad.mm(vec2.unsqueeze(-1).conj()).squeeze(-1)
```

Comment:
I had to modify this formula because outer supports half dtype right now but mv does not.

Reply:
We should probably add a comment and file an issue about replacing the formula with mv if/when it ever gets half support...

Are there any performance concerns here? I'm not too worried about 2 extra views here and I've rarely seen ger/outer used in the wild

Reply:
Actually due to more issues with cuda devices, I'll completely remove that formula and let this be autodiffed.
Note that outer was already autodiffed so that shouldn't change anything for this one.
And ger is deprecated so people should not be using it.